### DISCUSSION TITLE
Entity.findByName is not a function

### DISCUSSION THREAD

**Discussion by jasonxs45** [galacean/discussion#100] [Engine] [answered]
0.9文档里有提到全局查找实体的方法Entity.findByName，实际使用时报错Entity.findByName is not a function，离谱

**Comment by eyworldwide** [comment, marked as answer]
`findByName` 是一个实例方法，不是静态方法，应该是 ` entity.findByName`。文档有点误导，我们纠正一下。

**Comment by eyworldwide** [comment]
@jasonxs45 已修复，感谢反馈！https://github.com/galacean/galacean.github.io/pull/771
